Tools/c-analyzer/c_parser/preprocessor/__init__.py

```diff
@@ -168,9 +170,17 @@ def handling_errors(ignore_exc=None, *, log_err=None):
 }
 
 
+def _get_default_compiler():
```

Comment:
Minimalized ported version from https://github.com/python/cpython/blob/f4c03484da59049eb62a9bf7777b963e2267d187/Lib/distutils/ccompiler.py#L937-L956

https://github.com/python/cpython/blob/f4c03484da59049eb62a9bf7777b963e2267d187/Lib/distutils/ccompiler.py#L937-L956

Reply:
It may be worth having a short comment indicating this was derived from the distutils code.